src/modules/Auth/components/SignInForm.tsx

```diff
@@ -90,22 +91,25 @@ export const SignInForm = () => {
                   />
                 )}
               />
-              <Box alignSelf="flex-end">
+              <Box
+                display="flex"
+                justifyContent="space-between"
+                flexDirection="row"
+              >
+                <Button
+                  variant="text"
+                  type="button"
+                  onClick={() => history.push('/signup')}
```

Comment:
It's not the best choice to specify the route here as just a string. Please move it to the constant above and think perhaps you already have this route somewhere in your codebase, and you can reuse it.

Reply:
Where place file with constants?